tests/integration/test_integration.py

```diff
@@ -750,16 +751,14 @@ def test_span_tags(encoding, monkeypatch):
     monkeypatch.setenv("DD_TRACE_API_VERSION", encoding)
 
     t = Tracer()
-    with mock.patch("ddtrace.internal.writer.writer.log") as log:
+    with snapshot_context(token="tests.integration.test_integration.test_span_tags"):
         s = t.trace("operation", service="my-svc")
         s.set_tag("env", "my-env")
         s.set_metric("number", 123)
         s.set_metric("number", 12.0)
         s.set_metric("number", "1")
```

Comment:
wait, this test doesn't make any sense... we use the same metric name for all three.....

can you update this?